nipype/pipeline/engine/tests/test_join.py

```diff
@@ -640,14 +641,11 @@ def sq(x):
 def test_join_nestediters(tmpdir):
     tmpdir.chdir()
 
-    def exponent(x, p):
-        return x**p
-
     wf = pe.Workflow("wf", base_dir=tmpdir.strpath)
 
     xs = pe.Node(IdentityInterface(["x"]), iterables=[("x", [1, 2])], name="xs")
     ps = pe.Node(IdentityInterface(["p"]), iterables=[("p", [3, 4])], name="ps")
-    exp = pe.Node(Function(function=exponent), name="exp")
+    exp = pe.Node(Function(function=operator.pow), name="exp")
```

Comment:
I don't think the `Function` interface can handle builtin functions.

Reply:
I see, that would explain this error message, right?
```
TypeError: module, class, method, function, traceback, frame, or code object was expected, got builtin_function_or_method
```
The problem is that [`operator.pow()`](https://docs.python.org/3/library/operator.html#operator.pow) is part of the Python library, it is not documented to be a built-in function. I guess it's an undocumented alias for the built-in [`pow()`](https://docs.python.org/3/library/functions.html#pow).